modules/infra/cli/iface.c

```diff
@@ -327,12 +327,43 @@ static cmd_status_t iface_list(const struct gr_api_client *c, const struct ec_pn
 	return CMD_ERROR;
 }
 
+static cmd_status_t iface_stats(const struct gr_api_client *c, const struct ec_pnode * /*p*/) {
+	struct gr_infra_iface_stats_get_req req;
+	struct gr_infra_iface_stats_get_resp *resp = NULL;
+	cmd_status_t status = CMD_ERROR;
+	void *resp_ptr = NULL;
+	int ret;
+
+	// Send the new API request and wait for the response
+	ret = gr_api_client_send_recv(c, GR_INFRA_IFACE_STATS_GET, sizeof(req), &req, &resp_ptr);
+	if (ret < 0) {
+		errorf("failed to get interface stats: %s", strerror(-ret));
+		goto end;
+	}
+
+	resp = resp_ptr;
+	// Iterate through the key-value pairs in the response and print them
+	for (uint16_t i = 0; i < resp->n_stats; i++) {
+		printf("  \"%s\": %lu\n", resp->stats[i].name, resp->stats[i].value);
```

Comment:
Why add double quotes here? Please use a "scol" table like other commands with one line per interface and 4 columns for rx_packets, rx_bytes, tx_packets, tx_bytes.

```
grout# show interface stats
INTERFACE  RX_PACKETS     RX_BYTES  TX_PACKETS     TX_BYTES
p0           15125012  42512047533   149102654  43527153209
gr-loop0            5         3207          12        13254
p1          149102654  43527153209    15125012  42512047533
```